Dockerfile

```diff
@@ -1,27 +1,47 @@
-FROM python:3.8-slim
-
-RUN apt-get update && apt-get -y install git gcc
-
-RUN useradd -r app_user
-RUN mkdir /app
-RUN chown app_user /app/
-WORKDIR /app
+FROM python:3.10-alpine
+
+ENV PYTHONPATH="/usr/local/src/idunn/src"
+ENV PYTHONUNBUFFERED=1
```

Comment:
As I understand it, it is recommended to disable output buffers in docker images to avoid issues where a log line is not outputted before a pod crashes (I'm not sure how you can do this without raising a Python exception but I'll trust the community on this one) :+1: